jamventProto/jamctrllib/JamCtrlGasMix.cpp

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Carly,

I think this looks good.  Just some comments more on format and conventions.

So some of these parameters are output or input/output parameters.
I think is good doxygen convention to comment them is the following.

 * @param  - default is this is an input parameter
 * @param[out] - output parameter
 * @param[in, out] - input/output parameter
Says he who has been kind of sparse on this comments.

I don't think we don't need to pass all these pointers. Just pass the value for input, and use a reference for output or input/output.

const float * const oxyPercDesired_fracDec,
->
 * @param reqAirTopUpPressure_bar ...
const float oxyPercDesired_fracDec,

and
float * const reqAirTopUpPressure_bar)
->
 * @param[out] reqAirTopUpPressure_bar ...
float& reqAirTopUpPressure_bar